apisupport/apisupport.ant/src/org/netbeans/modules/apisupport/project/ui/customizer/SuiteCustomizerLibraries.java

```diff
@@ -780,7 +780,6 @@ public ExplorerManager getExplorerManager() {
         DISABLED_PLATFORM_MODULES.add("org.netbeans.modules.favorites"); // NOI18N
         // And these are deprecated:
         DISABLED_PLATFORM_MODULES.add("org.openide.compat"); // NOI18N
-        DISABLED_PLATFORM_MODULES.add("org.openide.util.enumerations"); // NOI18N
```

Comment:
The `apisupport` can work with older versions of the platform (that contain the  module). It may be wiser to keep this line.

Reply:
Agreed.. It is now fixed..